src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java

```diff
@@ -83,45 +83,41 @@
 	/*
 	 * SERVICES
 	 */
-	private ReferenceFileService referenceFileService;
-	private SequencingObjectService sequencingObjectService;
-	private AnalysisSubmissionService analysisSubmissionService;
-	private ProjectService projectService;
-	private UserService userService;
-	private IridaWorkflowsService workflowsService;
-	private MessageSource messageSource;
+	private final ReferenceFileService referenceFileService;
+	private final SequencingObjectService sequencingObjectService;
+	private final AnalysisSubmissionService analysisSubmissionService;
+	private final ProjectService projectService;
+	private final UserService userService;
+	private final IridaWorkflowsService workflowsService;
+	private final MessageSource messageSource;
 	private final WorkflowNamedParametersService namedParameterService;
-	private UpdateSamplePermission updateSamplePermission;
-	private AnalysisSubmissionSampleProcessor analysisSubmissionSampleProcessor;
-	private GalaxyToolDataService galaxyToolDataService;
-	private EmailController emailController;
-
-	/*
-	 * CONTROLLERS
-	 */
-	private CartController cartController;
+	private final UpdateSamplePermission updateSamplePermission;
+	private final AnalysisSubmissionSampleProcessor analysisSubmissionSampleProcessor;
+	private final GalaxyToolDataService galaxyToolDataService;
+	private final EmailController emailController;
+	private final UICartService cartService;
 
 	@Autowired
 	public PipelineController(SequencingObjectService sequencingObjectService,
 			ReferenceFileService referenceFileService, AnalysisSubmissionService analysisSubmissionService,
 			IridaWorkflowsService iridaWorkflowsService, ProjectService projectService, UserService userService,
-			CartController cartController, MessageSource messageSource,
+			MessageSource messageSource,
 			final WorkflowNamedParametersService namedParameterService, UpdateSamplePermission updateSamplePermission,
```

Comment:
Remove `final`

Reply:
This is an old relic, I cannot remember you used to like that.  Removed in 746b003e81aa0d6e9e5f6b072670b1c231c0d8fc